src/java/org/apache/cassandra/repair/RepairSession.java

```diff
@@ -467,6 +466,11 @@ public void onIRStateChange(LocalSession session)
         }
     }
 
+    public boolean accordOnly()
+    {
+        return repairData && repairAccord && !repairPaxos;
```

Comment:
!repairData?

Reply:
Looking at call site, I think it is the name that perhaps needs revisiting. `eligibleForAccordMigration()`?

Reply:
I have to page this back in. Things got weird when repairs of Accord at QUORUM instead of ALL were added.
I tried to disable it, but then there were test failures in `AccordIncrementalRepairTest`.

Data not being repaired at ALL for Accord creates correctness issues for things like migration as well as bulk reader.

I’ll revisit this.

Reply:
OK, so paging this back in.

In the beginning Accord barriers needed to be highly available for key barriers so they occurred at QUORUM and for repair that means you need the subsequent data repair to ensure the Accord data propagated at QUORUM was also at ALL. This is needed for offline bulk reading of repaired data at ONE and anything that tries to leverage that guarantee.

Then came the addition of `RepairSyncPointAdapter` which made the Accord repairs occur at ALL. And then things got confusing because for a given set of repair options what is supposed to be selected for a given repair and is it eligible for consensus migration varies more.

I'm going to do another round of cleanup so that we can tell if the Accord repair was eligible (whether it was at QUORUM + data repair or at ALL with no data repair) and we will use the more efficient barrier at ALL for migration (with no data repair) or consensus only repair. For a non-migration regular repair (accord + data) we can use QUORUM since the data repair will finish it anyways (either with or without down nodes depending on force).